index.js

```diff
@@ -146,4 +146,8 @@ searchBtnCmp.addEventListener("click", function() {
             console.log(backgroundImageUrl);
             document.getElementsByClassName("leftContainer")[0].style.backgroundImage = `url(${backgroundImageUrl})`;
         })
-});
\ No newline at end of file
+});
+
+function resetValues() {
+
```

Comment:
One suggestion how you can do it:
for example for Max-Temp: field- you can have in you html file structure like this:
`<p id="weatherMaxTemp">Max-Temp: <p id="maxTempValue"></p> &#8451</p>`
and then in your main function set value not as `appendChild` but just a value inside this new paragraph:
```js
  let newMaxTempValue = document.getElementById("maxTempValue");
  newMaxTempValue.innerHTML = Math.round(data.main.temp_max);
```

The difference is that you now working with one element all the time and setting it's inside value instead of adding more and more child elements upon user click on a button.

You can follow this logic for all other elements as well. So you not append childs all the time but set values.